### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by earl-nicholson**

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -15,35 +15,142 @@
...
+// create a function
+// use shift to remove the first item in an array
+// use sort to shuffle the array after removing the first item
+// look up different ways to use sort to shuffle the array
+
+const newArray = (array) => {
+  array.shift()
+  array.sort((a, b) => 0.6 - Math.random())
+  return array
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:48]: Awesome work! Why did you use .6 instead of .5? Just wondering.

File: code-challenges.test.js [hunk 2]
```diff
@@ -15,35 +15,142 @@
...
+// using both methods at the same time didnt work, to fix i seperated them and did two pushes first min and then max
+
+
+const minMax = (array) => {
+  let newArray = []
+  newArray.push(Math.min(...array))
+  newArray.push(Math.max(...array))
+  return newArray
+}
+
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:99]: Good job! I bet you can make this pseudo code even more language agnostic though...

File: code-challenges.test.js [hunk 3]
```diff
@@ -15,35 +15,142 @@
...
 // b) Create the function that makes the test pass.
+
+// create a funstion called removeD
+// use the spread operator to combine the two arrays
+// use new Set method to remove duplicates
+
+const removeD = (array1, array2) => {
+  array1 = [...new Set([...array1, ...array2])]
+  return array1
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:142]: Brilliant

File: ruby-challenges.rb [hunk 4]
```diff
@@ -10,6 +10,18 @@
...
+    "#{number} is even"
+  else
+    "#{number} is odd"
+  end
+end
+
+p evenOdd num1
+p evenOdd num2
+p evenOdd num3
+
```
Comment (WalkerTexas-coder) [inline ruby-challenges.rb:24]: Good work on all the ruby questions.

File: interview-questions.md [hunk 5]
```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 5. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: Implicit return means typing return is not required and is automatically ran.
 
-  Researched answer:
+  Researched answer: Implicit return means typing return is not required and is automatically ran on the last line of code
```
Comment (WalkerTexas-coder) [inline interview-questions.md:44]: Getting an interview question "correct" and being a compelling interviewee are two different things. These answers will not get you hired. They do not communicate an expertise or an experience with any of these materials. They don't show a curiosity or and understanding of the underlying concepts beyond what can be googled.

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
Everything except your interview questions looks amazing. Add two sentences per question to your interview questions and I will approve. 

I recommend talking about a time you used the tools being questioned and ran into something you didn't expect.

File: code-challenges.test.js (hunk 1 above)
Comment (earl-nicholson) [reply code-challenges.test.js:48]: .6 came from me playing with the code to get a better understanding on how it worked.

**Review by WalkerTexas-coder — APPROVED** [review]
After talking this over with Sarah I realize that this assessment does meet the bare minimum requirements. So I am approving this assessment. That said I highly suggest you take some time with this interview questions and put some deep effort into them. They may get a you a job one day.

**Pull request merged**